lsp-mode.el

```diff
@@ -1911,20 +1917,27 @@ WORKSPACE is the workspace that contains the progress token."
 (defun lsp--modeline-build-code-actions-string (actions)
   "Build the string to be presented on modeline for code ACTIONS."
   (-let* ((icon (lsp--modeline-code-actions-icon))
-          (first-action-string (propertize (->> actions
-                                                lsp-seq-first
-                                                lsp:code-action-title
-                                                (replace-regexp-in-string "[\n\t ]+" " "))
+          (first-action-string (propertize (or (-first #'lsp:code-action-is-preferred? actions)
```

Comment:
the first leg of the or will return CodeAction, the second leg will return string.

Reply:
good catch, done!